src/elements/content-sidebar/stories/tests/MetadataSidebarRedesign-visual.stories.tsx

```diff
@@ -358,9 +388,86 @@ export const MetadataInstanceEditorAIEnabled: StoryObj<typeof MetadataSidebarRed
         expect(autofillWithBoxAI).toHaveLength(2);
 
         const editButton = await canvas.findByRole('button', { name: 'Edit My Template' });
-        userEvent.click(editButton);
+        await userEvent.click(editButton);
 
         const autofillButton = await canvas.findByRole('button', { name: 'Autofill' });
         expect(autofillButton).toBeInTheDocument();
     },
 };
+
+export const ShowErrorWhenAIAPIIsUnavailable: StoryObj<typeof MetadataSidebarRedesign> = {
+    args: {
+        features: {
+            ...mockFeatures,
+            'metadata.aiSuggestions.enabled': true,
+        },
+    },
+    parameters: {
+        ...defaultVisualConfig.parameters,
+        msw: {
+            handlers: [
+                ...defaultMockHandlers,
+                http.post('https://api.box.com/2.0/ai/extract_structured', () => {
+                    return new HttpResponse('Internal Server Error', { status: 500 });
+                }),
+            ],
+        },
+        test: {
+            dangerouslyIgnoreUnhandledErrors: true,
+        },
+    },
+    play: async ({ canvasElement }) => {
+        const canvas = within(canvasElement);
+
+        const autofillButton = await canvas.findByRole(
+            'button',
+            { name: 'Autofill My Template with Box AI' },
+            { timeout: 5000 },
```

Comment:
do we really need 5s ? same for the other ones?

Reply:
I've changed that to waiting for side panel to be available

Reply:
Correction, I've changed timeout to `2000`, it works locally. 
Waiting for container was failing on re-runs.